docs/static/troubleshooting.asciidoc

```diff
@@ -73,18 +73,8 @@ Try adding these values to the `jvm.options` file.
 
 [source,sh]
 -----
---add-opens=java.base/java.lang=ALL-UNNAMED 
---add-opens=java.base/java.security=ALL-UNNAMED 
---add-opens=java.base/java.util=ALL-UNNAMED 
---add-opens=java.base/java.security.cert=ALL-UNNAMED 
---add-opens=java.base/java.util.zip=ALL-UNNAMED 
---add-opens=java.base/java.lang.reflect=ALL-UNNAMED 
---add-opens=java.base/java.util.regex=ALL-UNNAMED 
---add-opens=java.base/java.net=ALL-UNNAMED 
---add-opens=java.base/java.io=ALL-UNNAMED 
---add-opens=java.base/java.lang=ALL-UNNAMED
---add-opens=java.base/javax.crypto=ALL-UNNAMED
---add-opens=java.management/sun.management=ALL-UNNAMED
+--add-opens=java.base/java.security=ALL-UNNAMED
+--add-opens=java.base/java.io=ALL-UNNAMED
```

Comment:
think we'll need merge with these: https://github.com/jruby/jruby/blob/9.2.12.0/bin/.jruby.module_opts

```
--add-opens java.base/java.io=org.jruby.dist
--add-opens java.base/java.nio.channels=org.jruby.dist
--add-opens java.base/sun.nio.ch=org.jruby.dist
--add-opens java.management/sun.management=org.jruby.dist
```

Reply:
that raises an interesting point, even the two we listed could have a finer grained opens:

```
# https://github.com/colinsurprenant/jruby-stdin-channel/issues/1
--add-opens=java.base/java.io=com.jrubystdinchannel
# jruby-openssl
--add-opens=java.base/java.security=org.jruby.ext.openssl

```

Reply:
that won't work unless the dependencies are properly modularized with a [module-info](https://github.com/headius/invokebinder/blob/invokebinder-1.11/src/main/java/module-info.java) but even if they were there's still the problem of these being JRuby extensions and the .jar would likely need to be moved to a 'proper' module-path instead of relying on JRuby's custom class-loader to do the loading, 
we need to stick with `org.jruby.dist` and probably even "un-named" for the extensions, at least for now : 

```
--add-opens=java.base/java.io=ALL-UNNAMED
--add-opens=java.base/java.security=ALL-UNNAMED
--add-opens java.base/java.nio.channels=org.jruby.dist
--add-opens java.base/sun.nio.ch=org.jruby.dist
--add-opens java.management/sun.management=org.jruby.dist
```

Reply:
got it, thanks for the explanation, I've addressed this :+1: